packages/state/source/transaction-validator.ts

```diff
@@ -8,7 +8,7 @@ export class TransactionValidator implements Contracts.State.TransactionValidato
 	private readonly handlerRegistry!: Contracts.Transactions.ITransactionHandlerRegistry;
 
 	@inject(Identifiers.WalletRepository)
-	@tagged("state", "blockchain")
+	@tagged("state", "copy-on-write")
```

Comment:
Double check if this tag should be **clone**.

Reply:
Yes, or rather it depends.

At the moment the transaction validator is only used by the pool, which is only supposed to apply transactions on the copy-on-write wallet repo. Ideally, this would be tagged dynamically, but I couldn't make it work.

Reply:
Nevermind my previous comment.  Should be `clone`